README.md

```diff
@@ -5,6 +5,7 @@
 [![Slurp - Read the Docs](https://img.shields.io/website?label=read%20the%20docs&url=https%3A%2F%2Fslurp.readthedocs.io%2F)](https://slurp.readthedocs.io/)
 [![Preston Worker](https://img.shields.io/website?label=workers.dev&url=https%3A%2F%2Fpreston.slurp.workers.dev%2F)](https://preston.slurp.workers.dev)
 [![Magic Worker](https://img.shields.io/website?label=workers.dev&url=https%3A%2F%2Fmagic.slurp.workers.dev%2F)](https://magic.slurp.workers.dev/)
+[![Discord](https://img.shields.io/website?label=discord&url=https%3A%2F%2Fdiscord.gg%2FPW3Fh9Knpp)](https://discord.gg/PW3Fh9Knpp)
```

Comment:
The Discord button should be like how it is on `One` with the User online count

https://github.com/One-Language/One/blob/main/README.md